client/QuickStart/Utils.cs

```diff
@@ -139,13 +139,17 @@ public static async Task ExecuteAsync(IDisposable client, string configDatabaseN
                     {
                         var clientRequestProperties = CreateClientRequestProperties("CS_SampleApp_ControlCommand");
                         ICslAdminProvider adminClient = (ICslAdminProvider)client;
-                        result = (await adminClient.ExecuteControlCommandAsync(configDatabaseName, command, clientRequestProperties)).ToJObjects().ToArray();
+                        var disposableResult = (await adminClient.ExecuteControlCommandAsync(configDatabaseName, command, clientRequestProperties));
```

Comment:
use a 'using' statement
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement

Reply:
He can't actually, it's a feature of c# 8, and .net framework's max version is 7.3.

So we need to move this project to .net core for this.